### ISSUE TITLE
COAM\ Earned Credits - External EC request denied

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-coam#61]
#### Description:

An operator has their EC issuance request denied.

#### Acceptance Criteria:
As an operator, given that the request for EC issuance for an operation was denied, when they look at the 'Track Status of Issuance' page, then they will see that the request was denied, a reason for the denial from the director, and how to get more information and appeal.

Display:

- [ ] Text indicating the operator can email ghgregulator@gov.bc.ca for more information.

- [ ] Number of credits requested
- [ ] Status of issuance (denied)
- [ ] The reason for the denial (text from the CAS director field)
- [ ] BCCR Trading name
- [ ] BCCR holding account number

- [ ] Text indicating that if the operator wants to appeal this decision they can submit an appeal through the EAB (include a link to the [website](https://www.bceab.ca))

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

#### Definition of Ready (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

#### Definition of Done (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By/Blocking
Blocked by #59 

#### Notes:

-

**Comment by NicoleGovvy** [comment]
Confirm with the business that:

1. The text from the director is visible for operators
2. They want the ghgregulator email address here to get more information

**Comment by vesselak** [comment]
Wireframe developed and presented to business area. Moving ticked in backlog for development.
